coursedata/adventures/en.yaml

```diff
@@ -1528,6 +1602,28 @@ adventures:
                         print 'Great! You survived!'
                     ```
                 start_code: "print 'Escape from the haunted house!'"
+    piggybank:
+        name: Piggy bank
+        description: Count your pocket money!
+        image: hauntedhouse.png
+        default_save_name: Piggy Bank
+        levels:
+            11:
+                story_text: |
+                    ## Piggy Bank
+                    In this adventure you learn how to make a digital piggy bank, to calculate how much money you have and how long you need to save up to buy what you want!
+
+                    ## Example Hedy Code
+                    ```
+                    print 'The digital piggy bank'
+                    wish is ask 'What would you like to buy?'
+                    price is ask 'How much does that cost?'
+                    saved is ask 'How much money have you saved already?'
+                    allowance is ask 'How much pocket money do you get per week?'
+                    tosave is price - saved
```

Comment:
@MarleenGilsing Ik zou het mooier vinden als we hier to_save gebruiken, dat is meer "programmeerderig"

Reply:
Done